vendor/github.com/jacobsa/gcloud/gcs/read.go

```diff
@@ -138,6 +149,32 @@ func (b *bucket) NewReader(
 	return
 }
 
+
+// Custom function made to create a new reader using Storage Client Library
+func NewReaderSCL(
+        ctx context.Context,
+        req *ReadObjectRequest,bucketName string) (rc io.ReadCloser, err error){
+        // initialising the bucket
+        client, err := storage.NewClient(ctx)
+        if err != nil {
+                err = fmt.Errorf("Error in creating the client")
+                return
+        }
+        fmt.Println("New Storage Client Created for: ", req.Name)
+        bkt := client.Bucket(bucketName) //check if the bucket name is valid or not
+        obj:=bkt.Object(req.Name) //check if the object name is valid or not
```

Comment:
space before and after := .. please run gofmt or follow the style guide of this file/repo

Reply:
yeah, will do that